<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogElectrique</class>
 <widget class="QDialog" name="DialogElectrique">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>590</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QChartView" name="graphCourant">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="graphTension">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="graphPuissance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>361</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QChartView" name="graphEnergie">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCourant">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTension">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPuissance">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEnergie">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>540</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
